frontend/components/bmm/BmmTrackView.vue

```diff
@@ -11,10 +11,11 @@ const dateString = (date: Date) => {
 </script>
 
 <template>
-    <p class="flex cursor-pointer gap-2 rounded bg-slate-200 pl-2">
+    <div class="flex w-full cursor-pointer gap-2 rounded bg-slate-200 pl-2">
         <span v-if="track && track.publishedAt">
             {{ dateString(track.publishedAt.toDate()) }}</span
         >
-        <span class="rounded-r bg-slate-50 px-2">{{ track.title }}</span>
-    </p>
+        <span class="bg-slate-50 px-2 flex-grow">{{ track.title }}</span>
+        <span class="rounded-r text-lg pr-2"><template v-for="l in track.languages?.Languages"><span :title="l.code">{{l.flagEmoji}}</span></template> </span>
```

Comment:
You don't need to wrap the inner span in a template tag, you can simply put the `v-for` on the span :) Other than than, looks good

Reply:
Would it not repeat the span then?

Reply:
It does repeat the span now as well, the `template` element is looped for each language. 
However, [the template tag is not rendered](https://vuejs.org/api/built-in-special-elements.html#template), so you could just put the v-for on the direct child of the template element, and it would work the same

Using template elements in loops is often done to prevent wrapping everything in `div`s everywhere you loop over something

Reply:
Aaah, I missunderstood. I see what you mean now. The inner span came later when I added the tooltip and I forgot about the template. Thanks.